.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,40 @@
+name: Bug report
+description: Report a bug. For security vulnerabilities see Report a security vulnerability in the templates.
+title: "BUG:"
```

Comment:
Could you add a space after the colon for all of these?  Just to increase the chance that the user has it in there.